### ISSUE TITLE
Offset Module - Formula Suggestion

### ISSUE THREAD

**Issue by musicmaster88** [bogaudio/BogaudioModules#160]
To start, I love the Offset module. It's a nice, compact module that can easily manipulate CV's.  But it currently adds the offset first and then applies the scale. This leads to the offset also being scaled and the output signal not where you expect it to be. Would it be possible to change the order around; doing scaling first and then the offset? This would lead to the resulting signal where you would expect it to be.

**Comment by Petervos2018** [comment]
Maybe a rightclick option:
Scale->Offset
Offset->Scale(old behaviour)

**Comment by mdemanett** [comment]
Agreed that would be better.  And yes probably needs to be an option to avoid potentially breaking people's patches.  I'll take a look.

**Comment by mdemanett** [comment]
A new [test release](https://github.com/bogaudio/BogaudioModules/releases/tag/v1.1.35-test3) is available with this change.

I did add a menu option to control how it behaves -- but the new behavior (scale then offset) is the default, including in existing patches.  I looked for a way to have the new behavior be the default when new modules are added, while keeping the old for modules in existing patches, but didn't find a good way to do it reliably.  Where the change does mess up a patch, the old behavior can be restored with the menu option.
